### ISSUE TITLE
AutoEat wont eat on 2b2t

### ISSUE THREAD

**Issue by ThebesAndSound** [fr1kin/ForgeHax#147]
Switches to food and starts to eat, won't complete eating ever.

**Comment by fr1kin** [comment]
Yeah it's a strange issue. It will eventually finish eating but it takes longer than normal. Probably has something to do with NCP because it works on a vanilla server just fine.

**Comment by ThebesAndSound** [comment]
Yeah works fine in vanilla, but as of a couple days ago after a restart and apparent plugin update forgehax autoeat will not eat at all ever, worked intermittently before.

**Comment by fr1kin** [comment]
I've been told that this is a problem even if you try to manually eat. @Bestle perhaps you have something to do with this pozz?

**Comment by fr1kin** [comment]
🤔

**Comment by 0-x-2-2** [comment]
no comment

**Comment by MMK21Hub** [comment]
It could be to do with low TPS?

**Comment by fr1kin** [comment]
no its the anticheat

**Comment by fghsgh** [comment]
I can confirm this is also not working on a different anarchy server, with good TPS.
